### ISSUE TITLE
6-Test set présentation des résultats

### ISSUE THREAD

**Issue by ThomasLEBRET** [ThomasLEBRET/IAS-combatPokemon#6]
Pensez à regarder les différentes mesures de scores envisagées au départ sur l’ensemble de validation (une fois vos hyper-paramètres fixés). Vérifiez qu’il n’y a pas un gros problème que vous n’auriez pas vu. Si les résultats sont raisonnablement satisfaisants, allez y avec le test set ! (Bref, ne trichez pas lors du test !)

Lorsque vous présenterez vos résultats (mesures de score, performances), vous mettrez en avant les perfor- mances sur le test set, mais il est bon de les accompagner des performances mesurées sur le validation set, et sur le train set, pour permettre de comparer. (toutes choses égales par ailleurs, évidemment, donc avec les mêmes hyper-paramètres)

Pour l’analyse plus qualitative des prédictions, certaines visualisations ne permettent pas de présenter simultanément plusieurs résultats (par exemple si ce sont des images reconstruites, etc). Dans ce cas, privilégiez toujours le test set.

Ne vous focalisez pas seulement sur la performance quantitative (la valeur du score), essayez d’analyser la façon dont les résultats dépendent de vos choix (même si le niveau total n’est “pas très bon”). Évidemment, si vous voulez présenter l'impact d'un hyper-paramètre sur le score, logiquement, il faut le faire sur le validation set (en tout cas, observer ça sur le test set n'autorise pas à changer d'avis sur les hyper-paramètres
